script/input/1/common.json

```diff
@@ -0,0 +1,6 @@
+{
+    "ilk": "ALLOCATOR-SPARK-1",
+    "usdcUnitSize": 100,
+    "usdsUnitSize": 100,
+    "admin": "0x6F3066538A648b9CFad0679DF0a7e40882A23AA4"
```

Comment:
This is unused and shouldn't be set to a deployer address anyways.

Reply:
Removed